src/renderers/native/ReactNativeFiberErrorDialog.js

```diff
@@ -43,6 +48,7 @@ function ReactNativeFiberErrorDialog(capturedError: CapturedError): boolean {
   }
 
   const newError = new errorType(errorMessage);
+  (newError: any).framesToPop = framesToPop;
   newError.stack = errorStack;
```

Comment:
This is not fully equivalent because debug tooling and such can read the internal slot of this instead of the property name.